lib/CL/devices/basic/basic.c

```diff
@@ -368,6 +368,7 @@ pocl_init_cpu_device_infos (cl_device_id dev)
   dev->llvm_cpu = get_llvm_cpu_name ();
 #endif
 
+  dev->spirv_version = "SPIR-V_1.2";
```

Comment:
Why does the spirv_version include the prefix "SPIR-V_" ?

Reply:
The `clGetDeviceInfo` spec says so: 

> Returns a space-separated list of IL version strings of the form <IL_Prefix>_<Major_Version>.<Minor_Version>. For OpenCL 2.1, "SPIR-V" is a required IL prefix.

Reply:
ok, didn't know it was standardized